reginald/models/models/llama_index.py

```diff
@@ -344,6 +347,7 @@ def _load_turing_way(self, gh_token: str) -> None:
             repo=repo,
             verbose=False,
             concurrent_requests=1,
+            timeout=180,
```

Comment:
shall we change this back to 60 like the others?

Reply:
Yeah I reckon so, 60 seems like a good no.